components/src/core/DropdownToolbar/DropdownToolbar.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useCallback, useRef } from 'react';
+import { Typography, Toolbar, ListItemText, Menu, MenuItem, ToolbarProps, createStyles, makeStyles, useTheme, Theme } from '@material-ui/core';
+import { ArrowDropDown } from '@material-ui/icons';
+import clsx from 'clsx';
+
+const MENU_ITEM_HEIGHT = 48;
+
+export type ToolbarMenuItem = {
+  label: string;
+  onClick: Function;
+};
+export type DropdownToolbarProps = ToolbarProps & {
+  title: string;
+  subtitleLabel: string | undefined;
+  menuItems?: ToolbarMenuItem[] | undefined;
```

Comment:
```suggestion
  subtitleLabel?: string;
  menuItems?: ToolbarMenuItem[];
```

Reply:
subtitleLabel should probably be required?

And maybe it makes sense to have the subtitleLabel taking a react node so that our users can pass in whatever they feel like rendering (like a channel value)? @joebochill

Reply:
Yeah I think ReactNode would be appropriate...we'd need to test it out to make sure it looks okay with some basic things like string, icon, channelvalue, <Link>. Other than that I think we can leave the user on their own.

I'm also fine with leaving it as a string for the first pass (if we don't want to put in the extra styling) since that will be the most common case.